tutorials/get-started/index.md

```diff
@@ -0,0 +1,1245 @@
+
+          final index = i ~/ 2;
+          if (index >= _suggestions.length) {
+            _suggestions.addAll(generateWordPairs().take(10));
+          }
+          return _buildRow(_suggestions[index]);
+        });
+  }
+
+  Widget _buildRow(WordPair pair) {
+    final alreadySaved = _saved.contains(pair);
+    return new ListTile(
+        title: new Text(
+          pair.asPascalCase,
+          style: _biggerFont,
+        ),
+        trailing: new Icon(
+          alreadySaved ? Icons.favorite : Icons.favorite_border,
+          color: alreadySaved ? Colors.red : null,
+        ),
+        onTap: () {
+          setState(() {
+            if (alreadySaved) {
+              _saved.remove(pair);
+            } else {
+              _saved.add(pair);
+            }
+          });
+        });
+  }
+}
+{% endprettify %}
+
+---
+
+## Step 6: Navigate to a new screen
+
+In this step, you'll add a new screen (called a _route_ in Flutter) that
+displays the favorites. You'll learn how to navigate between the home route
+and the new route. Finally, you'll add a Theme to the app's UI.
+
+In Flutter, the Navigator manages a stack containing the app's routes.
+Pushing a route onto the Navigator's stack, updates the display to that route.
+Popping a route from the Navigator's stack, returns the display to the previous
+route.
+
+<ol markdown="1">
+<li markdown="1">
+Add a list icon to the AppBar in the `build` method for RandomWordsState.
+When the user clicks the list icon, a new
+route that contains the favorites items is pushed to the Navigator, displaying the
+new list.
+
+Note that the `actions` property takes an array of widgets. Some properties
+take a single widget (`child`), and other properties take multiple widgets
+(`children`), as indicated by the square brackets (`[]`).
+
+Add the highlighted "action" code to the build method:
+
+<!-- skip -->
+{% prettify dart %}
+class RandomWordsState extends State<RandomWords> {
+  ...
+  @override
+  Widget build(BuildContext context) {
+    return new Scaffold(
+      appBar: new AppBar(
+        title: new Text('Startup Name Generator'),
+        [[highlight]]actions: <Widget>[[[/highlight]]
+          [[highlight]]new IconButton([[/highlight]]
+            [[highlight]]icon: new Icon(Icons.list),[[/highlight]]
+            [[highlight]]onPressed: _pushSaved[[/highlight]]
+          [[highlight]])[[/highlight]]
+        [[highlight]]][[/highlight]]
+      ),
+      body: _buildSuggestions(),
+    );
+  }
+  ...
+}
+{% endprettify %}
+</li>
+
+<li markdown="1">
+Add a `_pushSaved()` function to the RandomWordsState class.
+
+<!-- skip -->
+{% prettify dart %}
+class RandomWordsState extends State<RandomWords> {
+  ...
+  void _pushSaved() {
+  }
+}
+{% endprettify %}
+
+Run the app. The list icon appears in the app bar. Tapping it does nothing
+yet, because the `_pushSaved` function is empty.
+</li>
+
+<li markdown="1">
+When the user taps the list icon in the app bar, build a Material route
```

Comment:
Do we need to say something about the "Material" in "Material route" here? If it's not important, we can just say "route".

Reply:
Good point. I deleted "Material".